src/pkg/mcp/tools/deploy.go

```diff
@@ -0,0 +1,156 @@
+		deployResp, project, err := cli.ComposeUp(ctx, project, grpcClient, provider, compose.UploadModeDigest, defangv1.DeploymentMode_DEVELOPMENT)
+		if err != nil {
+			err = fmt.Errorf("failed to compose up services: %w", err)
+			logger.Sugar.Errorw("Failed to compose up services", "error", err)
+
+			return mcp.NewToolResultText(fmt.Sprintf("Failed to compose up services: %v", err)), nil
+		}
+
+		if len(deployResp.Services) == 0 {
+			logger.Sugar.Errorw("Failed to deploy services", "error", errors.New("no services deployed"))
+			return mcp.NewToolResultText(fmt.Sprintf("Failed to deploy services: %v", errors.New("no services deployed"))), nil
+		}
+
+		// Get the portal URL for browser preview
+		portalURL := printPlaygroundPortalServiceURLs(deployResp.Services)
+
+		// Open the portal URL in the browser if available
+		if portalURL != "" {
+			logger.Sugar.Infof("Opening portal URL in browser: %s", portalURL)
+			go func() {
+				err := browser.OpenURL(portalURL)
+				if err != nil {
+					logger.Sugar.Errorw("Failed to open URL in browser", "error", err, "url", portalURL)
+				}
+			}()
+		}
+
+		// Success case
+		logger.Sugar.Infow("Successfully started deployed services", "etag", deployResp.Etag)
+
+		// Create a simple output for the tool result
+		var output strings.Builder
+
+		// Log deployment success
+		logger.Sugar.Info("Deployment Started!")
+		logger.Sugar.Infof("Deployment ID: %s", deployResp.Etag)
+
+		// Add minimal information to output for tool result
+		output.WriteString("Deployment Started. See logs tool for details.")
+
+		// Log browser preview information
+		logger.Sugar.Infof("🌐 %s available", portalURL)
+
+		// Log service details
+		logger.Sugar.Info("Services:")
+		for _, serviceInfo := range deployResp.Services {
+			logger.Sugar.Infof("- %s", serviceInfo.Service.Name)
+			logger.Sugar.Infof("  Public URL: %s", serviceInfo.PublicFqdn)
+			logger.Sugar.Infof("  Status: %s", serviceInfo.Status)
+		}
+
+		// Return the etag data as text
+		return mcp.NewToolResultText(fmt.Sprintf("Please use the web portal url: %s to follow the deployment of %s, with the deployment ID of %s", portalURL, project.Name, deployResp.Etag)), nil
+	})
+}
+
+const DEFANG_PORTAL_HOST = "portal.defang.io"
+const SERVICE_PORTAL_URL = "https://" + DEFANG_PORTAL_HOST + "/service"
+
+// printPlaygroundPortalServiceURLs logs service URLs for the Defang portal
+// and returns the first URL for browser preview
+func printPlaygroundPortalServiceURLs(serviceInfos []*defangv1.ServiceInfo) string {
+	// Log portal URLs for monitoring services
+	logger.Sugar.Info("Monitor your services' status in the defang portal")
+
+	// Store the first URL to return for browser preview
+	var firstURL string
+
+	for _, serviceInfo := range serviceInfos {
+		serviceURL := SERVICE_PORTAL_URL + "/" + serviceInfo.Service.Name
+		logger.Sugar.Infof("   - %s", serviceURL)
+
+		// Save the first URL we encounter
+		if firstURL == "" {
+			firstURL = serviceURL
+		}
```

Comment:
Can we print all of the urls instead of just the first one? The user may have multiple accessible services.